tsconfig.json

```diff
@@ -1,4 +1,9 @@
 {
+  "include": [
+    "packages/create-react-native-library/src",
+    "packages/create-react-native-library/package.json",
```

Comment:
`import { version as crnlVersion } from '../package.json';` This line was generating errors during typecheck without including.